zone/mob.cpp

```diff
@@ -4000,20 +4000,26 @@ uint8 Mob::GetDefaultGender(uint16 in_race, uint8 in_gender) {
 	}
 }
 
-void Mob::SendAppearancePacket(uint32 type, uint32 value, bool WholeZone, bool iIgnoreSelf, Client *specific_target) {
+void Mob::								SendAppearancePacket(uint32 type, uint32 value, bool WholeZone, bool iIgnoreSelf, Client *specific_target, bool WholeZoneMinusMyGuild) {
```

Comment:
???

Reply:
Same here.  Like it had a party or something....

Updated